secret.go

```diff
@@ -16,17 +17,25 @@ import (
 
 func applySecret(ctx *cli.Context) error {
 	baseURL := getNGBaseURL(ctx)
-	gitPat := ctx.String("token")
+	token := ctx.String("token")
 	filePath := ctx.String("file")
-	gitPat = getGitSecret(gitPat)
+	gitPat := getGitSecret(token)
 	secretType := ctx.String("secret-type")
+	authType := ctx.String("auth-type")
+	port := ctx.String("port")
+	username := ctx.String("username")
+	domain := ctx.String("domain")
 	requiresFile := isFileTypeSecret(secretType)
 	secretIdentifier := getSecretIdentifier(secretType)
 	secretName := getSecretName(secretType)
 	var secretBody HarnessSecret
-	var headers map[string]string
 	var err error
 
+	portNumber, portErr := strconv.Atoi(port)
+
+	if portErr != nil {
```

Comment:
is this a required check or was just for troubleshooting purposes?

Reply:
required check as port number is a number